configuration/security.md

```diff
@@ -92,6 +92,30 @@ alice ALL=(ALL) PASSWD: ALL
 
 Now save the file.
 
+
+## Disable or enable root login
+
+To disable root login, use:
```

Comment:
@spl237 @XECDesign  Doesn't Raspbian already come without a root login? If so, should this section be removed?